### PR TITLE
Add GCB staging bucket for CRI-O

### PR DESCRIPTION
**Pull request by saschagrunert**
Initiated by https://github.com/kubernetes/test-infra/issues/15569

Our plan is to store test results in this world-readable bucket.

### CODE REVIEW

**Comment by dims** [comment]
/hold

looks like conformance buckets are being worked on in https://github.com/kubernetes/k8s.io/pull/459

**Comment by thockin** [comment]
Is this about conformance or staging for artifacts and images?

**Comment by saschagrunert** [comment]
> Is this about conformance or staging for artifacts and images?

It’s more about prow test artifacts for CI/conformance. :)

**Comment by stp-ip** [comment]
Then it should be "conformance" instead of "staging" in https://github.com/kubernetes/k8s.io/pull/501/files#diff-8b242825ad9027b6efead6a412418c51R937

The difference is conformance buckets are much closer to production buckets. Aka their data is held for much longer and anything in there is not something considered temporary.
Staging might not live more than a few months so test results comparison would be problematic.

For context:
staging = short lived, testing stuff, no expectation on retention
conformance =  long lived, test results etc, long retention
production = long lived, general production and not special case such as conformance, long retention

**Comment by thockin** [comment]
There's a different script now for conformance buckets - check out `ensure-conformance-storage.sh`

**Comment by saschagrunert** [comment]
Alright thanks for the wrap-up. I changed it as suggested, PTAL :)

File: groups/groups.yaml [hunk 1]
```diff
@@ -994,3 +994,24 @@ groups:
       - ihor.dvoretskyi@gmail.com
       - jdumars@gmail.com
       - jorge@heptio.com
+
+  - email-id: k8s-infra-conformance-cri-o@kubernetes.io
```
Comment (stp-ip) [inline groups/groups.yaml]:
```suggestion
  - email-id: k8s-infra-conform-cri-o@kubernetes.io
```

File: groups/groups.yaml [hunk 2]
```diff
@@ -994,3 +994,24 @@ groups:
...
+    name: k8s-infra-conformance-cri-o
```
Comment (stp-ip) [inline groups/groups.yaml]:
```suggestion
    name: k8s-infra-conform-cri-o
```

Comment (saschagrunert) [reply groups/groups.yaml]: Fixed that as well :+1:

File: groups/groups.yaml (hunk 1 above)
Comment (saschagrunert) [reply groups/groups.yaml]: Fixed as suggested :+1:

**Comment by stp-ip** [comment]
/lgtm

Waiting for hold being lifted from @dims and/or @thockin

**Comment by thockin** [comment]
Actuating now

**Comment by saschagrunert** [comment]
@dims may I ask you how to access this bucket now? I would expect that I have access to the `k8s-conform` project in Google Cloud but this is unfortunately not the case. :thinking:

**Comment by dims** [comment]
bucket seems to be `gs://k8s-conform-cri-o/` @saschagrunert 

```
dims@dims-a01:~/go/src/k8s.io/k8s.io$ infra/gcp/ensure-conformance-storage.sh cri-o
Ensuring project exists: k8s-conform
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-conform/billingInfo
projectId: k8s-conform
Enabling the GCS API
Configuring conformance bucket for cri-o
Configuring bucket: gs://k8s-conform-cri-o
Ensuring the bucket exists and is world readable
Enabling Bucket Policy Only for gs://k8s-conform-cri-o...
No changes made to gs://k8s-conform-cri-o/
Ensuring the GCS bucket retention policy is set: k8s-conform
Setting Retention Policy on gs://k8s-conform-cri-o/...
Empowering GCS admins
Updated IAM policy for project [k8s-conform].
bindings:
- members:
  - serviceAccount:service-228988630781@compute-system.iam.gserviceaccount.com
  role: roles/compute.serviceAgent
- members:
  - serviceAccount:228988630781-compute@developer.gserviceaccount.com
  - serviceAccount:228988630781@cloudservices.gserviceaccount.com
  role: roles/editor
- members:
  - user:thockin@google.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  role: roles/viewer
etag: BwWfZa0EzCo=
version: 1
No changes made to gs://k8s-conform-cri-o/
No changes made to gs://k8s-conform-cri-o/
Empowering k8s-infra-conform-cri-o@kubernetes.io to GCS
No changes made to gs://k8s-conform-cri-o/
No changes made to gs://k8s-conform-cri-o/
Done
dims@dim
```

**Comment by bartsmykla** [comment]
@saschagrunert can you check if you have access to the bucket @dims wrote about? If not let me know, I'm gonna try help. :-)

**Comment by saschagrunert** [comment]
> @saschagrunert can you check if you have access to the bucket @dims wrote about? If not let me know, I'm gonna try help. :-)

Hey, I have access to the bucket. But right now I'm trying to get the service account key for usage in an external CI system. See https://github.com/kubernetes/test-infra/issues/15081#issuecomment-590890421 and https://kubernetes.slack.com/archives/CCK68P2Q2/p1582649587028000)

**Pull request merged**